modules/standard/ChapelEnv.chpl

```diff
@@ -33,7 +33,7 @@ program with the ``--about`` flag.
 
 .. warning::
 
-    This module has been deprecated - please use :mod:`Subprocess` instead.
+    This module has been deprecated - please use :mod:`ChplConfig` instead.
```

Comment:
Well, that's an embarrassing mistake on my part...  Weirdly, I thought I'd discovered and fixed this myself before now.  Maybe it got orphaned somehow?